chrome_agent/js/extension/background.js

```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+__axiomExport__(this);
+var chromeAgent = new chrome_agent.ChromeAgent();
```

Comment:
Sounds good, except that this is doing the opposite (it is the agent, so it is listening for a connection from Axiom).  Also, I realized that because a listener can be connected to multiple times, this shouldn't be a promise, so I switched it to an event system (onConnected).